README.md

```diff
@@ -332,6 +333,54 @@ Currently specifying endpoints is not supported through the extras field.
 If in your rtsp_address your hostname is UPNP_DISCOVER then we will try to find a UPnP host that matches.
 You can filter the results by filling out the `query` field in the configuration. See `viamupnp.DeviceQuery` for supported filters.
 
+## Configure the `viamrtsp:unifi` discovery service
+
+This model is used to discover RTSP cameras connected to a [UniFi Protect](https://ui.com/camera-security) NVR. It uses the UniFi Protect Integration API to enumerate cameras and retrieve their RTSP stream URLs.
+
+```json
+{
+   "nvr_address": "<NVR_IP_OR_HOSTNAME>",
+   "unifi_token": "<API_TOKEN>"
+}
+```
+
+### Attributes
+
+| Name    | Type   | Inclusion    | Description |
+| ------- | ------ | ------------ | ----------- |
+| `nvr_address` | string | **Required** | The IP address or hostname of the UniFi Protect NVR (e.g., `"10.1.14.106"`). |
+| `unifi_token` | string | **Required** | API token for authenticating with the UniFi Protect NVR. Generate this in UniFi Protect under Settings > API Token. |
```

Comment:
should we add an external link to instructions on their docs?

Reply:
Added link to docs:
https://developer.ui.com/site-manager-api/gettingstarted#obtaining-an-api-key